visits/admin.py

```diff
@@ -117,12 +118,30 @@ class ParticipationAdmin(admin.ModelAdmin):
                     'accepted', 'present')
     list_filter = ('submitted', 'accepted', 'present')
     actions = [accept_selected_participations, reject_selected_participations]
-    def user_link (self,book):
-        url=reverse("admin:users_user_change", args=[book.user.id]) 
+
+    def user_link(self, book):
+        url = reverse("admin:users_user_change", args=[book.user.id])
         link = '<a href="%s">%s</a>' % (url, book.user.email)
         return mark_safe(link)
     user_link.short_description = 'Utilisateur'
 
+    actions = ["export_as_csv"]
+
+    def export_as_csv(self, request, queryset):
+        meta = self.model._meta
+        field_names = [field.name for field in meta.fields]
+
+        response = HttpResponse(content_type='text/csv')
+        response['Content-Disposition'] = 'attachment; filename={}.csv'.format(
+            meta)
+        writer = csv.writer(response)
+        writer.writerow(field_names)
+        for obj in queryset:
+            row = writer.writerow([getattr(obj, field)
+                                   for field in field_names])
+        return response
+    export_as_csv.short_description = "Exporter au format CSV"
```

Comment:
Cette fonction n'a pas l'air d'être différente de celle pour `UserAdmin`. Une possibilité ce serait de créer une [classe mixin](https://stackoverflow.com/questions/533631/what-is-a-mixin-and-why-are-they-useful), de ce genre :

```python
class CSVExportMixin:

    model = None
    
    def export_as_csv(self, request, queryset):
        meta = self.model._meta
        # ... mettre le contenu de l'action ici ...

    def get_actions(self, request):
        # on récupère les actions par défaut
        # (une liste vide si `actions` n'est pas défini)
        actions = super().get_actions(request)
        # on ajoute notre action d'export
        actions.append('export_as_csv')
        return actions
```

(Voir [get_actions()](https://docs.djangoproject.com/fr/2.1/ref/contrib/admin/actions/#django.contrib.admin.ModelAdmin.get_actions).)

Cette classe pourrait être dans `core/mixins.py` par exemple (vu que c'est réutilisable).

Et on l'utiliserait comme ça:

```python
class ParticipationAdmin(CSVExportMixin, ModelAdmin):
    # ... le reste de la classe, sans ce qui est déjà défini par CSVExportMixin
```